### PR TITLE
fix: [TERR-316] - Fix for non-rendering of edge data rates in tooltip

### PR DESCRIPTION
**Pull request by sanchezelton**
There was disagreement in the code (likely a typo) as to whether to use double or triple hyphens in the dataParser module. Prior changes in 2021 had apparently been done to change notation for edge endpoints to align with what was being reported in ESDB at the time (the use case however, is still unknown). However, the convention remains to use double hyphens. 

This PR refactors the dataParser by storing the delimiter in its own variable and updates occurrences of double or triple hyphen in the dataParser module. Corresponding generated code in dist, as well as plugin.json, have been updated as well.

There are also some spelling corrections in README.md and the generated options.d* files have been readded back into the dist directory.

### Checklist

-   [x] PR Title format: `type: [TERR-XXX] Short description`
    -   `type` is one of: 'build', 'ci', 'chore', 'docs', 'feat', 'fix', 'perf', 'refactor', 'revert', 'style', 'test'
    -   `TERR-XXX` refers to a Jira issue.
    -   If this is a breaking change prefix the description with "BREAKING CHANGE:"
-   [x] There is a related Jira issue for this pull request
-   [x] Description should be a meaningful summary of the changes you are proposing
-   [x] For breaking changes prefix the title with "BREAKING CHANGE:" and include details in the description
-   [x] This PR is as small as possible
-   [x] This PR includes tests related to these changes, or existing tests provide coverage
-   [x] This PR has updated documentation as appropriate

### Notes

-   Once approved, use the **squash and merge** option.
-   Refer to the [development guidelines](https://github.com/esnet/esnet-portal/blob/develop/docs/development/developer_tips.md) for more details.

### CODE REVIEW

File: src/components/lib/dataParser.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { DataFrameView } from '@grafana/data';
 
+const EDGE_ENDPT_DELIMITER = '--';
```
Comment (jkafader-esnet) [inline src/components/lib/dataParser.js:3]: `EDGE_DELIMITER` or `ENDPOINT_DELIMITER`

File: dist/options.d.ts
Comment (jkafader-esnet) [inline dist/options.d.ts:1]: Why is this file created? I thought this was options.js in the last go round?

**Review by jkafader-esnet — APPROVED** [review]

File: src/components/lib/dataParser.js (hunk 1 above)
Comment (sanchezelton) [reply src/components/lib/dataParser.js]: Updated.

File: dist/options.d.ts
Comment (sanchezelton) [reply dist/options.d.ts:1]: This is an autogenerated file created by tsc from a file that was originally name src/options.ts, then renamed src/options.js in order to allow a JS module esmap to import values exported from options.js.
The TS config has a flag, `allowJs`, that generates type descriptors based on all compiled files, both TS and JS.

**Pull request merged**